Boolder/Models/Problem.swift

```diff
@@ -244,9 +244,13 @@ extension Problem {
     }
     
     var next: Problem? {
-        if let circuitNumberInt = Int(self.circuitNumber), let circuitId = circuitId {
-            let nextNumber = String(circuitNumberInt + 1)
-            
+        var nextNumber = "1";
+        
+        if let circuitNumberInt = Int(self.circuitNumber) {
+            nextNumber = String(circuitNumberInt + 1)
+        }
+
+        if let circuitId = circuitId {
```

Comment:
maybe we could use `guard let` to make the code a bit easier to read?
I find it harder to read when there is nested code